includes/class-wc-stripe-order-handler.php

```diff
@@ -23,12 +23,12 @@ public function __construct() {
 
```

Comment:
To avoid the prefixed with an underscore warning, I've seen the usage of $instance instead of $_this.